src/PowerShell/Microsoft.WinGet.Client/README.md

```diff
@@ -6,6 +6,15 @@ This project contains the source code for building the Windows Package Manager P
 
 After building the project solution, the `Microsoft.WinGet.Client` PowerShell module can be found in the output directory in the `PowerShell` folder. For example if you built the project as x64 release, you should expect to find the module files in `$(SolutionDirectory)/src/x64/Release/PowerShell`.
 
+## Prerequisites
+
+The PowerShell module has a dependency on the Microsoft Visual C++ Redistributable package which can be downloaded [here](https://learn.microsoft.com/en-US/cpp/windows/latest-supported-vc-redist?view=msvc-170) or installed through WinGet by running the following command:

```

Comment:
What version do I need to download?

Reply:
@florelis I've got the [Microsoft Visual C++ 2015-2022 Redistributable, v14.34.31931](https://learn.microsoft.com/en-US/cpp/windows/latest-supported-vc-redist?view=msvc-170) on my W11 machine and it works for me (well, it imports and Get-WinGetPackage runs).

Reply:
I added the word "latest" and the id in the winget example points to 2015.

Reply:
Thanks @robinmalik. I actually meant it as a rhetorical question for @ryfu-msft to specify it in the doc, which he did